spec/units/git_repository_spec.rb

```diff
@@ -171,32 +171,43 @@
   end
 
   context '#config' do
-    it 'returns a git configuration value' do
-      with_a_temporary_home_directory do
-        in_a_temporary_directory do
-          repo = Gitsh::GitRepository.new(env)
-          run 'git init'
-          run 'git config --local alias.zecho "!echo zzz"'
-          expect(repo.config('alias.zecho')).to eq '!echo zzz'
+    context 'for a variable that is set' do
+      it 'returns a git configuration value' do
+        with_a_temporary_home_directory do
+          in_a_temporary_directory do
+            repo = Gitsh::GitRepository.new(env)
+            run 'git init'
+            run 'git config --local alias.zecho "!echo zzz"'
```

Comment:
would be helpful to make `"!echo zzz"` a variable here - would improve readability of test, in my opinion!